docs/docs/30-administration/22-backends/40-kubernetes.md

```diff
@@ -47,6 +47,19 @@ agent:
 
   Additional annotations to apply to worker pods. Must be a YAML object, e.g. `{"example.com/test-annotation":"test-value"}`.
 
+## Pipeline level configuration
```

Comment:
```suggestion
## Tips and tricks

### CRI-O
```

And I would move this section at the end of the document.

Reply:
Good idea. Done.